clang/lib/3C/DeclRewriter.cpp

```diff
@@ -376,7 +377,7 @@ void DeclRewriter::rewriteMultiDecl(DeclReplacement *N, RSet &ToRewrite,
 // invoking the rewriter) is to add any required initializer expression.
 void DeclRewriter::doDeclRewrite(SourceRange &SR, DeclReplacement *N) {
   std::string Replacement = N->getReplacement();
-  if (dyn_cast<TypedefDecl>(N->getDecl()))
+  if (isa<TypedefDecl>(N->getDecl()))
```

Comment:
Interesting. I wonder what the difference is, practically speaking.

Reply:
I think `isa` requires proper setup in the constructor?

Reply:
The only difference is that `dyn_cast` returns a nullable pointer and `isa` returns a boolean of whether the pointer would have been non-null, so this change is correct.  [Documentation](https://clang.llvm.org/extra/clang-tidy/checks/llvm-prefer-isa-or-dyn-cast-in-conditionals.html).